src/galario.cpp

```diff
@@ -1114,8 +1120,14 @@ void create_image_d(int nr, const dreal* const ints, dreal Rmin, dreal dR, int n
 
 void create_image_h(int const nr, const dreal *const ints, dreal const Rmin, dreal const dR, int const nxy, dreal const dxy,
                     dreal const inc, dcomplex *const image) {
+
     CPUTimer t;
 
+    dreal ratio_central_pixel = (dxy/2.-Rmin)/dR;
```

Comment:
DRY principle! Let's move this into a helper function called at from `create_image` and `create_image_d`